src/main/java/seedu/programmer/storage/JsonAdaptedStudent.java

```diff
@@ -115,7 +115,7 @@ public Student toModelType() throws IllegalValueException {
                 int labTotal = adaptedLab.getLabTotal();
 
 
-                if (!LabNum.isValidLabNum(labNum)) {
+                if (!LabNum.isValidLabNum(labNum) || labNum == 0) {
```

Comment:
Shouldn't labNum = 0 be considered invalid lab number already?

Reply:
<img width="406" alt="Screenshot 2021-11-08 123232" src="https://user-images.githubusercontent.com/77187494/140684781-e657cd98-e3ad-4f13-95b2-dcb50c9c8bf8.png">

I attempted to change `this.newLabNum = new LabNum(0)` to `this.newLabNum = new LabNum(original.getLabNumValue())` , but it  leads to unexpcted behaviour for editLabCommand and I cannot figure out the reason.

It seems like lab 0 is not allowed for addlab and editlab command now and this appears to be a quick fix for the manually editing data issue.

Reply:
I see, should we create an issue for this?

Reply:
It is definitely preferable to do so, but I am not sure if we have time to do major fix given that we still have to work on documentation

(I tried to solve in #587, but there is an issue with checking for duplicated labs and it might need a significant fix on how EditLabCommand works)

Reply:
Appreciate the bug fix from @reddevil1313 #591 , I can now fix the issue at isValidLabNum level.

Please review again and let me know if there are further issues!

Reply:
Thanks @ZhaoPeiduo and @reddevil1313 !